packages/editor/templates/Coin/README.md

```diff
@@ -235,7 +235,7 @@ function isValidAddress(address) {
 }
 
 function isValidAmount(amount) {
-    return amount > 0 && typeof Number(amount) == 'number';
+    return amount > 0 && typeof Number(amount) == 'number';    
```

Comment:
Same deal, I'd remove these trailing spaces.

Reply:
```suggestion
    return amount > 0 && typeof Number(amount) == 'number';
```

I have  a dodgy MBP keyboard and get random stuff everywhere right now :(